packages/trpc/server/routers/loggedInViewer/integrations.handler.ts

```diff
@@ -205,6 +95,8 @@ export const integrationsHandler = async ({ ctx, input }: IntegrationsOptions) =
         isSetupAlready,
         ...(app.dependencies && { dependencyData }),
       };
+
+      return appData;
     })
   );
 
```

Comment:
Was thinking of shifting all the code below into a function of its own, but don't think it'll be really helpful since the logic is fairly simple to understand(just an if conditional) and there's a lot of variables from input. What do you guys thinkabout this, should we split the code below into its own separate function?

Reply:
Probably, there's too many variables and if statements involved here not to.